bot/code_review_bot/tasks/lint.py

```diff
@@ -98,40 +106,25 @@ def as_markdown(self):
             line=self.line,
             message=self.message,
             publishable=self.is_publishable() and "yes" or "no",
-            disabled_rule=self.is_disabled_rule() and "yes" or "no",
+            disabled_check=self.is_disabled_check() and "yes" or "no",
             is_new=self.is_new and "yes" or "no",
         )
 
-    def as_dict(self):
+    def build_extra_informations(self):
         """
         Outputs all available information into a serializable dict
         """
-        return {
-            "analyzer": "mozlint",
-            "level": self.level,
-            "path": self.path,
-            "linter": self.linter,
-            "line": self.line,
-            "column": self.column,
-            "nb_lines": self.nb_lines,
-            "rule": self.rule,
-            "message": self.message,
-            "validation": {"disabled_rule": self.is_disabled_rule()},
-            "in_patch": self.revision.contains(self),
-            "is_new": self.is_new,
-            "validates": self.validates(),
-            "publishable": self.is_publishable(),
-        }
+        return {"linter": self.linter}
```

Comment:
Maybe in the mozlint case linter should be the analyzer name?

Reply:
definitely